src/app/content/components/PrevNextBar.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { ChevronLeft } from 'styled-icons/boxicons-regular/ChevronLeft';
+import { ChevronRight } from 'styled-icons/boxicons-regular/ChevronRight';
+import { linkStyle, textRegularStyle } from '../../components/Typography';
+import theme from '../../theme';
+import { AppState } from '../../types';
+import * as select from '../selectors';
+import { Book, LinkedArchiveTreeSection, Page } from '../types';
+import { findArchiveTreeSection } from '../utils/archiveTreeUtils';
+import ContentLink from './ContentLink';
+
+// tslint:disable-next-line:variable-name
+const LeftArrow = styled(ChevronLeft)`
+  height: 3rem;
+  width: 3rem;
+`;
+
+// tslint:disable-next-line:variable-name
+const RightArrow = styled(ChevronRight)`
+  height: 3rem;
+  width: 3rem;
+`;
+
+interface PropTypes {
+  page?: Page;
+  book?: Book;
+  prevNext?: {
+    prev?: LinkedArchiveTreeSection;
+    next?: LinkedArchiveTreeSection;
+  };
+}
+
+// tslint:disable-next-line:variable-name
+const PrevPage = styled.span`
+  ${textRegularStyle}
+  ${linkStyle}
+  border: none;
+`;
+
+// tslint:disable-next-line:variable-name
+const NextPage = styled.span`
+  ${textRegularStyle}
+  ${linkStyle}
+  border: none;
+`;
+
+// tslint:disable-next-line:variable-name
+const BarWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: ${theme.color.neutral.base};
```

Comment:
you probably don't need to specify the background color, i think the parent element does